packages/local_auth/local_auth_darwin/darwin/Classes/messages.g.m

```diff
@@ -41,7 +41,7 @@ - (instancetype)initWithValue:(FLADAuthResult)value {
 }
 @end
 
-/// Pigeon equivalent of the subset of BiometricType used by iOS.
+/// Pigeon equivalent of the subset of BiometricType used by iOS & macOS.
```

Comment:
Please see line 4 of this file; you are hand-editing a generated file rather than the source of generation.

Reply:
Reverted